tests/ignite/engine/test_create_supervised.py

```diff
@@ -96,7 +96,7 @@ def test_create_supervised_training_scalar_assignment():
     optimizer = SGD(model.parameters(), 0.1)
 
     with mock.patch("ignite.engine._check_arg") as check_arg_mock:
-        check_arg_mock.return_value = None, torch.cuda.amp.GradScaler(enabled=True)
+        check_arg_mock.return_value = None, torch.cuda.amp.GradScaler(enabled=False)
```

Comment:
@OBITORASU could you please explain this, why setting it as False ? Thanks

Reply:
Yes sure I set this one to False because of this error message in the logs `("torch.cuda.amp.GradScaler is enabled, but CUDA is not available. Disabling.")`. I assumed that this defaulted to False due to the lack of CUDA being available, I think this warning stopped showing up in the logs after flipping True to False as well.